src/Pages/Nav/nav.jsx

```diff
@@ -1,15 +1,14 @@
 import React from "react";
-import './nav.css'
+import "./nav.css";
 import { Link } from "react-router-dom";
 import logo from "../../Assets/logo.png";
 
-import {useAuth} from "../../Auth/auth-context"
+import { useAuth } from "../../Auth/auth-context";
 
 function Nav() {
+  const { isLoggedIn, logOut } = useAuth();
 
-  const {isLoggedIn,logOut} = useAuth()
-
-    // function to toggle on small screen
+  // function to toggle on small screen
   function ClickHandler() {
     const navList = document.getElementsByClassName("navbar-list")[0];
     return navList.classList.toggle("toggle-active");
```

Comment:
Suggestion: We should not use getElementsByClassName in React since it is against the declarative concept of React..